test/run_tests.bash

```diff
@@ -10,6 +10,13 @@
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 TEST_RESULTS_DIR="${SCRIPT_DIR}/results"
+mkdir -p "$TEST_RESULTS_DIR"
```

Comment:
Preferable to `.gitkeep` it or just make sure it exists? 

https://github.com/sea-bass/pyrobosim/actions/runs/7770678869/job/21191139631#step:5:10

Reply:
I think the `mkdir -p` approach is fine!